src/dataExplorer/components/DeleteScript.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {FC, useContext} from 'react'
+import {Button, ComponentColor, IconFont, Overlay} from '@influxdata/clockface'
+import './SaveAsScript.scss'
+import {CLOUD} from 'src/shared/constants'
+import {PersistanceContext} from 'src/dataExplorer/context/persistance'
+import {useHistory} from 'react-router-dom'
+import {useSelector, useDispatch} from 'react-redux'
+import {getOrg} from 'src/organizations/selectors'
+import {notify} from 'src/shared/actions/notifications'
+import {deleteScriptFail} from 'src/shared/copy/notifications/categories/scripts'
+
+let deleteScript
+
+if (CLOUD) {
+  deleteScript = require('src/client/scriptsRoutes').deleteScript
+}
+
+interface Props {
+  onBack: () => void
+  onClose: () => void
+}
+
+const OpenScript: FC<Props> = ({onBack, onClose}) => {
+  const {resource} = useContext(PersistanceContext)
+  const history = useHistory()
+  const dispatch = useDispatch()
+  const org = useSelector(getOrg)
+
+  const handleDeleteScript = async () => {
+    if (resource.data?.id) {
+      try {
+        const resp = await deleteScript({scriptID: resource.data.id})
+
+        if (resp.status !== 204) {
+          throw new Error(resp.data.message)
+        }
+
+        history.replace(`/orgs/${org.id}/data-explorer/from/script/`)
+        onClose()
+      } catch (error) {
+        dispatch(notify(deleteScriptFail(resource.data?.name)))
+      }
+    }
+  }
+  return (
+    <Overlay.Container maxWidth={500}>
+      <Overlay.Header
+        title="Are you sure you want to delete?"
+        onDismiss={onClose}
+      />
+      <Overlay.Body>
+        "{resource.data?.name}" will be deleted and cannot be undone.
+      </Overlay.Body>
+      <Overlay.Footer>
+        <Button
+          color={ComponentColor.Tertiary}
+          onClick={onClose}
+          text="cancel"
+        />
+        <Button color={ComponentColor.Secondary} onClick={onBack} text="back" />
+        <Button
+          color={ComponentColor.Danger}
+          onClick={handleDeleteScript}
+          text="delete"
+          icon={IconFont.Trash_New}
+        />
+      </Overlay.Footer>
+    </Overlay.Container>
+  )
+}
+
+export default OpenScript
```

Comment:
```suggestion
export {DeleteScript}
```
Same as above. And also it's a good time to use named exports, related to #6007